### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by tenyiting**
ready

### CODE REVIEW

**Comment by tenyiting** [comment]
ready

**Comment by ngocvt-0484** [comment]
please fix conflict

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  database: db/development.sqlite3
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
```
Comment (ngocvt-0484) [inline config/database.yml]: 1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
```git rm --cached config/database.yml```
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

**Comment by tenyiting** [comment]
ready

**Review by ghost — COMMENTED** [review]
cmt

**Comment by tenyiting** [comment]
ready

File: config/database.yml (hunk 1 above)
Comment (ngocvt-0484) [reply config/database.yml]: not fix yet

**Comment by ngocvt-0484** [comment]
please change 5 commits to 1 commit

File: config/database.yml (hunk 1 above)
Comment (ngocvt-0484) [reply config/database.yml]: this file will not be up to GIt, please
```cp database.yml database.yml.example
git rm --cached config/database.yml
```
and in .gitignore add
```config/database.yml```

**Comment by ngocvt-0484** [comment]
you can use git rebase -i 
readmore http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages

**Comment by tenyiting** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**